content/docs/user-guide/project-structure/dvcyaml-files.md

```diff
@@ -725,28 +725,30 @@ stages:
       feature: [feature1, feature2, feature3]
     cmd: ./train.py --feature ${item.feature} ${item.model}
     outs:
-      - ${item.model}.pkl
+      - ${key}.pkl # equivalent to: ${item.model}-${item.feature}.pkl
```

Comment:
Not a docs question, but does the core PR guarantee the ordering of keys (model first here) matches the way it's written in `dvc.yaml`?

Reply:
As Python dicts keep the ordering, the generated key should reflect the order of the matrix inputs: https://github.com/iterative/dvc/blob/main/dvc/parsing/__init__.py#L533